osu.Game/Overlays/MusicController.cs

```diff
@@ -179,7 +179,9 @@ public MusicController()
         protected override bool OnDrag(InputState state)
         {
             Vector2 change = (state.Mouse.Position - state.Mouse.PositionMouseDown.Value);
-            change.X = -change.X;
+
+            if ((Anchor & Anchor.x2) > 0)

```

Comment:
Let's make a second conditional for Anchor.y2. Or alternatively, should be stop reversing the x and y directions for certain anchors? Would certainly simplify a lot of code.